### PR TITLE
Model redesign

### PR DESCRIPTION
**Pull request by Miguel4president**
Resulting changes from the last PR.

### CODE REVIEW

File: src/Application/Models/Asset.js [hunk 1]
```diff
@@ -1,12 +1,58 @@
...
+    return this._multiplier;
+  }
+  get owned() {
+    return this._owned;
+  }
+  get unlocked() {
+    return this._unlocked;
+  }
+
+  makeBuy(increaseRatio) {
```
Comment (samgqroberts) [inline src/Application/Models/Asset.js]: `toBought()`

Comment (samgqroberts) [inline src/Application/Models/Asset.js]: nvm `buy()`

File: src/Application/Models/Bank.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  makeIncomeUpdate(income) {
+    return new Bank(
+        this._cash,
+        income,
+        this._total,
+        this._lastRentTime
+    );
+  }
+
+  makeBuy(cost, newIncome) {
```
Comment (samgqroberts) [inline src/Application/Models/Bank.js]: probably `withdraw()`

File: src/Application/Models/Broker.js [hunk 3]
```diff
@@ -0,0 +1,82 @@
+import _ from 'underscore';
+
+const PRICE_INCREASE_PERCENTAGE = 1.07;
```
Comment (samgqroberts) [inline src/Application/Models/Broker.js]: not a percentage, rather it's a multiplier i think

Comment (Miguel4president) [inline src/Application/Models/Broker.js]: ratio, changed it.

File: src/Application/Models/Bank.js (hunk 2 above)
Comment (Miguel4president) [inline src/Application/Models/Bank.js]: Renamed all of them

**Pull request merged**